web/components/bet-panel.tsx

```diff
@@ -215,7 +215,7 @@ function BuyPanel(props: {
 
   useEffect(() => {
     // warm up cloud function
-    placeBet({}).catch()
```

Comment:
These changes aren't strictly necessary for the operation of this PR, but I put them in anyway since it was convenient. All these warmup functions would actually blow up if an error happened (e.g. the function returns a 500), because `catch()` is a no-op -- you need an actual handler function in order for anything to get caught.

Reply:
Good catch (pun intended).